styles.css

```diff
@@ -394,7 +398,7 @@ li.frameworks-about p,
 li.skills-about p {
   padding-top: 6px;
   position: relative;
-  width: 327px;
+  width: auto;
```

Comment:
- [ ] Please kindly work on the design to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)
**Your Design**
![you](https://user-images.githubusercontent.com/55596970/187988026-d3d4989b-7645-4232-8d0f-da247f8adefd.png)
**Figmal Design**
![skill_Yours](https://user-images.githubusercontent.com/55596970/187988116-71984cd4-37c0-4125-93b9-63c56facbee3.png)